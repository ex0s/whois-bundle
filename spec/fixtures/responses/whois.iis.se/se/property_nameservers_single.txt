# Copyright (c) 1997- .SE (The Internet Infrastructure Foundation).
# All rights reserved.

# The information obtained through searches, or otherwise, is protected
# by the Swedish Copyright Act (1960:729) and international conventions.
# It is also subject to database protection according to the Swedish
# Copyright Act.

# Any use of this material to target advertising or
# similar activities is forbidden and will be prosecuted.
# If any of the information below is transferred to a third
# party, it must be done in its entirety. This server must
# not be used as a backend for a search engine.

# Result of search for registered domain names under
# the .SE top level domain.

# The data is in the UTF-8 character set and the result is
# printed with eight bits.

state:            active
domain:           nhv.se
holder:           nordis0702-00149
admin-c:          -
tech-c:           -
billing-c:        -
created:          1992-11-05
modified:         2014-03-18
expires:          2014-12-31
nserver:          freja.nhv.se 82.96.40.83
dnssec:           unsigned delegation
status:           ok
registrar:        SE Direkt
