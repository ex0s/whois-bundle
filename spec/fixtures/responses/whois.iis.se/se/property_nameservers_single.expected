#nameservers
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{nameserver}
  %s[0].name == "freja.nhv.se"
  %s[0].ipv4 == "82.96.40.83"
  %s[0].ipv6 == nil
